projects/portal/src/app/pages/faucet/faucet.usecase.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Config, ConfigService } from 'projects/portal/src/app/models/config.service';
+import { BehaviorSubject, combineLatest, Observable } from 'rxjs';
+import { first, map } from 'rxjs/operators';
+
+@Injectable({ providedIn: 'root' })
+export class FaucetUseCaseService {
+
+  //private config$: Observable<Config | undefined>;
+
+  constructor(
+    private readonly route: ActivatedRoute,
+    private configService: ConfigService,
+  ) {
+    //this.config$ = this.configService.config$;
+    //this.denom$ = new BehaviorSubject('');
+  }
+
+  get config$(): Observable<Config | undefined> {
+    return this.configService.config$;
+  }
```

Comment:
Thanks for splitting the config$!

Next up is an added bonus.
The final thing needed in the view is the `faucetURL` value. If we can create it up front in the UseCaseService, it will simplify the responsibilities.

(It would be good to start this modification after the UsecaseService has been split without error.)

https://github.com/UnUniFi/web-apps/blob/02deee0b294c9c3cf43fdf010e1f7ea6fd381bcf/projects/portal/src/app/views/faucet/faucet.component.ts#L51-L53

Reply:
fixed so that `faucetURL` is created in `pages/faucet/faucet.component.ts`